asyncdb/drivers/sqlite.py

```diff
@@ -160,7 +160,7 @@ async def fetch_many(self, sentence: str, size: int = None):
             if not self._result:
                 raise NoDataFound()
         except Exception as err:
-            error = "Error on Query: {err}"
+            error = f"Error on Query: {err}"
```

Comment:
it should be fine as it is since it is done this way every where, also for the other drivers, this would be a general improvement which should be addressed in a dedicated commit if needed.